CHANGELOG.md

```diff
@@ -17,7 +25,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
-- **Breaking**: Update to ESLint v7 (#46, #58, #62, #63)
+- **(SEMVER-MAJOR)** Update to ESLint v7 (#46, #58, #62, #63)
```

Comment:
I don't understand this change :thinking: I don't really know what "SEMVER-MAJOR" is meant to convey - "Breaking" seemed clearer to me.

Reply:
This is inspired by the Node.js changelog, e.g. [this one](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V14.md) (ctrl+f "semver-major"). Ironically, because I think it's clearer.

As for what it's meant to convey, it seems pretty self-explanatory to me. The changelog uses semantic versioning. This change alone necessitates a major version bump by the semver spec 🤷

Reply:
Well OK, but I don't really get it. All "Semver-major" conveys to me is that it breaks something, and it seemed more straightforward to just say what we mean. But I guess it doesn't matter.

They seem to use that term because they use the `semver-major` and `semver-minor` labels on PRs to decide what version number to use for each release, and to help generate the changelog (using the tool [changelog-maker](https://github.com/nodejs/changelog-maker)).

Reply:
I tend to agree that "Breaking" is clearer. I can see the benefit of automation if we were using automated changelog tools but from a quick grep perspective **Breaking** gets my attention more than **SEMVER-MAJOR**

Reply:
I dun did an allcaps `BREAKING:`. Good?

Reply:
BREAKING